app/pages/notifications/notification-section.jsx

```
@@ -0,0 +1,253 @@
+import React, { Component } from 'react';
+import Notification from './notification';
+import apiClient from 'panoptes-client/lib/api-client';
+import talkClient from 'panoptes-client/lib/talk-client';
+import { Link } from 'react-router';
+import Paginator from '../../talk/lib/paginator';
+import updateQueryParams from '../../talk/lib/update-query-params';
+import ZooniverseLogo from '../../partials/zooniverse-logo';
+
+class NotificationSection extends Component {
+  constructor(props) {
+    super(props);
+    this.onSectionToggle = this.onSectionToggle.bind(this);
+    this.state = {
+      currentMeta: { },
+      firstMeta: { },
+      lastMeta: { },
+      notificationsMap: { },
+      notifications: [],
+      page: 1,
+    };
+  }
+
+  componentWillMount() {
+    if (this.props.section === 'zooniverse') {
+      this.setState({
+        name: 'Zooniverse',
+      });
+    } else {
+      apiClient.type('projects').get({ id: this.props.projectID, cards: true })
+      .catch(() => {
+        return null;
```

Is this to just swallow errors silently? Not sure that's a great idea 😨